synapse/storage/database.py

```diff
@@ -541,6 +541,7 @@ def new_transaction(
                         },
                     ):
                         r = func(cursor, *args, **kwargs)
+                        opentracing.log_kv({"message": "commit"})
```

Comment:
Is the idea that we can now time how long the function vs commit takes? I think `start_active` span should tag the span with `error` if any of this fails?